### ISSUE TITLE
Commercial use issue

### ISSUE THREAD

**Issue by SimonHFL** [PrithivirajDamodaran/Gramformer#9]
Hey @PrithivirajDamodaran  

The readme states that Gramformer versions above 1.0 are allowed for commercial use - however, this is not currently the case as the [grammar_error_correcter_v1](https://huggingface.co/prithivida/grammar_error_correcter_v1) model has been trained using the non-commercial  [WI&Locness](https://huggingface.co/datasets/wi_locness) data, even though the documentation states otherwise: 

The  [grammar_error_correcter_v1](https://huggingface.co/prithivida/grammar_error_correcter_v1) model is actually identical to the previous [grammar_error_correcter](https://huggingface.co/prithivida/grammar_error_correcter)  model which is trained using the non-commercial WI&Locness data – they have identical weights, which you can verify with this [script](https://gist.github.com/SimonHFL/0ca08654a9ef87a10500a4141d7fd252) 

As the models are the same, this means that both models have been trained using the non-commercial [WI&Locness](https://huggingface.co/datasets/wi_locness) data, and the [grammar_error_correcter_v1](https://huggingface.co/prithivida/grammar_error_correcter_v1) model along with Gramformer v1.1 and v1.2 should not be allowed for commercial use. 

Could you please update the readme to clarify this, or upload a new model that has not been trained using WI&Locness?

Thanks

**Comment by PrithivirajDamodaran** [comment]
Hey @SimonHFL 

- As per our email exchange the pre-release model (prithivida/grammar_error_correcter) was trained using filtered WikiEdits data and on top of that, a slice of WI&Locness is used. Because at the time WI&Locness was available as a HuggingFace dataset with no license, in fact, marked as **"unknown"** (Below are the proofs for that, find attached the screenshots). I have already mentioned this information in the email thread, to which you said it was probably an unintentional miss at the end of the people who uploaded the dataset to HuggingFace. So, to reiterate no intention to undermine anyone's academic work or violate a valid license policy, I merely used it based on the license info shown at that point in time.

![Screenshot 2021-06-19 at 8 54 44 PM](https://user-images.githubusercontent.com/7071019/123640308-a869a580-d83e-11eb-8ffe-e2be35940ad5.png)
![Screenshot 2021-06-19 at 8 55 11 PM](https://user-images.githubusercontent.com/7071019/123640346-b3bcd100-d83e-11eb-9081-2d87439373e7.png)

- (I can see that you have/had them update the license info recently.)
- So after you pointed out a **possible gap/missing info on the license** on the HuggingFace page, I acknowledged that  in the email (also mentioned I am anyway in the process of gathering more WikiEdits data to train the subsequent models) and I did the following: A.) Explicitly called out the pre-release model is not intended for commercial usage in Github, B.) Did the same in HuggingFace readme and C.) Trained a brand new model excluding WI&Locness. That is the _V1 model.
-  _V1 model (prithivida/grammar_error_correcter_v1) is trained using [WikiEdit pairs](https://github.com/snukky/wikiedits) and other synthetic pairs (refer to the readme for details)
- Your script is saying both pre-release and V1 models are identical because there might be an inadvertent oversight on my side in picking the right checkpoints while uploading to the tag v1. 
- I have refreshed the v1 tag with the right checkpoint files now and double-checked. See below

![Screenshot 2021-06-29 at 7 37 14 AM](https://user-images.githubusercontent.com/7071019/123726058-e56c8100-d8ac-11eb-9ef4-a64fd2fcaf95.png)

- Also to avoid any future unintentional non-compliance in the usage from the consumers of the package <= v1.0 and hence the pre-release model (prithivida/grammar_error_correcter), I can remove it from HuggingFace.


Thanks
